dev-debug/apitrace/apitrace-13.0.ebuild

```diff
@@ -0,0 +1,110 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..14} )
+inherit cmake-multilib multilib optfeature python-single-r1
+
+DESCRIPTION="Tool for tracing, analyzing, and debugging graphics APIs"
+HOMEPAGE="https://github.com/apitrace/apitrace"
+# Newer than the submodule commit from 13.0, but this commit includes a patch for tests.
+# See #738888
+BACKTRACE_COMMIT="d4bdca038e8685af3043683a881af37489fc1827"
+SRC_URI="
+	https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/ianlancetaylor/libbacktrace/archive/${BACKTRACE_COMMIT}.tar.gz
+		-> ${P}-libbacktrace-${BACKTRACE_COMMIT}.tar.gz
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="gui test X"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+RESTRICT="!test? ( test )"
+
+DEPEND="${PYTHON_DEPS}
+	app-arch/brotli:=[${MULTILIB_USEDEP}]
+	app-arch/snappy:=[${MULTILIB_USEDEP}]
+	media-libs/libpng:0=
```

Comment:
Why isn't multilib libpng needed? This is suspicious given the only conditional parts for multilib are USE=gui. If it's only needed for the gui though, it should be folded into that conditional.

Reply:
It's required by gui and gltrace tools (multilib). Fixed.